modules/tracker/controllers/components/ApiComponent.php

```diff
@@ -661,8 +281,17 @@ private function _createOrFindByName($itemName, $community)
     /**
      * Create a new submission.
      *
+     * @param producerRevision The repository revision of the producer that produced this value
```

Comment:
I'm getting caught again on 'repository revision of the producer' and later 'the branch name within the source repository'.

This is probably on me to understand the data model better, but I'm noting it here in case I make a targeted pass later.